libraries/botbuilder/src/test/java/com/microsoft/bot/builder/core/adapters/TestFlow.java

```diff
@@ -2,18 +2,21 @@
 
 import com.microsoft.bot.builder.core.ServiceKeyAlreadyRegisteredException;
 import com.microsoft.bot.builder.core.TurnContext;
+import com.microsoft.bot.builder.core.TurnContextImpl;
 import com.microsoft.bot.schema.models.Activity;
 import org.joda.time.DateTime;
 
 import java.lang.management.ManagementFactory;
 import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
 import java.util.function.Function;
 
 import static java.util.concurrent.CompletableFuture.completedFuture;
+import static java.util.concurrent.CompletableFuture.supplyAsync;
 
 public class TestFlow {
     final TestAdapter adapter;
```

Comment:
should this be "_adapter" ?
(and consistently across the other member variables? and then not use the "this")

Reply:
Apparently, no _ and accessing via "this."